packages/shiki-twoslash/src/parseCodeFenceInfo.ts

```diff
@@ -34,7 +38,14 @@ export function parseCodeFenceInfo(lang: string, fullMetaString: string) {
     return fail(`Invalid character in language name: '${current()}'`)
   }
 
-  return { languageName, meta }
+  if (fullMetaString.match(titlePattern)) {
+    meta.title = fullMetaString.match(titlePattern)?.[2] ?? ""
+  }
+
+  return {
+    languageName,
+    meta,
+  }
```

Comment:
Seems reasonable to me, this will need to be documented somewhere in the shiki-twoslash README too

Reply:
I'll do this. Let me know when you're happy

Reply:
I'm finished. Feel free to take on. Was about to go to bed so good luck.